packages/payment-detection/src/near-info-retriever.ts

```diff
@@ -17,124 +21,45 @@ export class NearInfoRetriever {
     private paymentReference: string,
     private toAddress: string,
     private proxyContractName: string,
-    private procedureName: string,
     private eventName: PaymentTypes.EVENTS_NAMES,
     private network: string,
   ) {
     if (this.network !== 'aurora' && this.network !== 'aurora-testnet') {
       throw new Error('Near input data info-retriever only works with Near mainnet and testnet');
     }
-    this.nearWebSocketUrl = NEAR_WEB_SOCKET_URL;
+    if (this.network !== 'aurora') {
```

Comment:
If we keep aurora as the name of the chain, why not naming the subgraph aurora?

Reply:
Hm good point. Overall I'm super confused about which name to use where. And yes it should actually be the time to remove autobahn finally !!!

Reply:
In the end, the chain should be renamed Near instead of Aurora, but I prefere making it a different topic 👌 @benjlevesque